### PR TITLE
Upgrade to GitLab CE 15.3.5

### PR DESCRIPTION
**Pull request by antt1995**

### CODE REVIEW

File: Changelog.md [hunk 1]
```diff
@@ -2,6 +2,11 @@
 
 This file only reflects the changes that are made in this image. Please refer to the upstream GitLab [CHANGELOG](https://gitlab.com/gitlab-org/gitlab-foss/blob/master/CHANGELOG.md) for the list of changes in GitLab.
 
+**15.3.5**
+
+- gitlab: upgrade CE to v15.3.5
+- gitaly: upgrade to v15.3.5
+
```
Comment (sachilles) [inline Changelog.md:9]: Could you be so kind and add

    ubuntu: upgrade to focal-20220922

(See my comment related to the Dockerfile.)

File: Dockerfile [hunk 2]
```diff
@@ -1,14 +1,14 @@
 FROM ubuntu:focal-20220826
```
Comment (sachilles) [inline Dockerfile:1]: Could you be so kind and use the base image ubuntu:focal-20221019? (Just to reduce the docker image size.)

**Review by sachilles — CHANGES_REQUESTED** [review]
Please check my comments. Thank you in advance.

**Comment by antt1995** [comment]
@sachilles As Requested

**Review by sachilles — APPROVED** [review]
LGTM! Thanks.

**Pull request merged**